docs/pages/distribution/app-stores.md

```diff
@@ -1,81 +1,37 @@
+This guide offers best practices around submitting your app to the app stores. To learn how to generate native binaries for submission, refer to ["Creating your first build"](/build/setup/).
 
 > **Disclaimer:** Review guidelines and rules are updated frequently, and enforcement of various rules can sometimes be inconsistent. There is no guarantee that your particular project will be accepted by either platform, and you are ultimately responsible for your app's behavior. That said, you can re-submit your app as needed to address feedback from reviews.
 
-## Prerequisites
-
-<Collapsible summary="Apple Developer Program membership is required to distribute to the Apple App Store.">
-
-You will need an Apple account with a $99 USD [Apple Developer Program](https://developer.apple.com/programs) membership in order to submit your app to the Apple App Store.
-
-</Collapsible>
-
-<Collapsible summary="Google Play Developer membership is required to distribute to the Google Play Store.">
+<BoxLink title="Versioning your app" description="Learn how to configure native runtime versions for your apps." href="/build-reference/app-versions" />
+<BoxLink title="App Store presence" description="Manage your Apple App Store metadata from the command line." href="/eas-metadata/introduction" />
+<BoxLink title="Permissions" description="Refine native permissions and system dialog messages." href="/guides/permissions" />
+<BoxLink title="App icons" description="App stores have strict rules for home screen icons." href="/guides/app-icons" />
+<BoxLink title="Splash screen" description="Create a seamless loading experience using with the splash screen API." href="/guides/splash-screens" />
+<BoxLink title="Status bar" description="Configure the native status bar to have high contrast on all screens." href="/guides/configuring-statusbar" />
+<BoxLink title="Apple: Review guidelines" description="Official Apple guide on preparing your app for App Store review." href="https://developer.apple.com/app-store/review" />
 
-You will need a Google account with a [Google Play Developer membership](https://play.google.com/apps/publish/signup/), which can be purchased for a one-time $25 USD fee.
-
-</Collapsible>
-
-## Make sure your app works on many form factors
+## Responsive design
 
 It's a good idea to test your app on a device or simulator with a small screen (e.g. an iPhone SE) as well as a large screen (e.g. an iPhone X). Ensure your components render the way you expect, no buttons are blocked, and all text fields are accessible.
```

Comment:
```suggestion
It's a good idea to test your app on a device or simulator with a small screen (for example, an iPhone SE) and a large screen (for example, an iPhone X). Ensure your components render the way you expect, no buttons are blocked, and all text fields are accessible.
```

(Unrelated to the changes to the, PR so far but I think this is nice to have improvement)
Let's avoid Latin abbreviations.